packages/manager/src/components/TypeToConfirm/TypeToConfirm.tsx

```diff
@@ -0,0 +1,106 @@
+import * as React from 'react';
+import Typography from 'src/components/core/Typography';
+import TextField from 'src/components/TextField';
+import Link from 'src/components/Link';
+import { makeStyles, Theme } from 'src/components/core/styles';
+import PreferenceToggle, { ToggleProps } from 'src/components/PreferenceToggle';
+import Grid from 'src/components/Grid';
+import CheckBox from 'src/components/CheckBox';
+import FormControlLabel from 'src/components/core/FormControlLabel';
+
+export interface Props {
+  confirmationText?: JSX.Element | string;
+  onChange: (str: string) => void;
+  label: string;
+  hideDisable?: boolean;
+  hideLabel?: boolean;
+  textFieldStyle?: Record<string, any>;
+  typographyStyle?: Record<string, any>;
```

Comment:
I think this would more accurate

```suggestion
  textFieldStyle?: React.CSSProperties;
  typographyStyle?: React.CSSProperties;
```